installers-conda/resources/post-install.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -i
 set -e
```

Comment:
@ccordoba12, this should do the trick.
```suggestion
set -e
unset HISTFILE
```

Reply:
Yep, I think so too.